tlslite/constants.py

```diff
@@ -553,6 +553,14 @@ class CipherSuite:
     TLS_EMPTY_RENEGOTIATION_INFO_SCSV = 0x00FF
     ietfNames[0x00FF] = 'TLS_EMPTY_RENEGOTIATION_INFO_SCSV'
 
+    # TLS 1.3 ciphersuites
+    TLS_AES_128_GCM_SHA256 = 0x1301
+    ietfNames[0x1301] = 'TLS_AES_128_GCM_SHA256'
+    TLS_AES_256_GCM_SHA384 = 0x1302
+    ietfNames[0x1302] = 'TLS_AES_256_GCM_SHA384'
+    TLS_CHACHA20_POLY1305_SHA256 = 0x1303
+    ietfNames[0x1303] = 'TLS_CHACHA20_POLY1305_SHA256'
+
```

Comment:
yes, it's intentional. tlslite-ng does not support CCM and I don't always add all the code points - it's done arbitrarily "when I need it"